Drakhtar/Managers/SDLAudioManager.h

```diff
@@ -47,9 +48,15 @@ class SDLAudioManager final : public AudioManager {
   void fadeOutChannel(int channel, int ticks);
   void fadeOutMusic(int ticks);
 
+  // check if its muted
+  bool getMuted();
+  void setMute(bool mute);
+  void checkMuted();
```

Comment:
This should be private, as it's only used internally.

Reply:
It is called in the OptionsMenu button when clicked, so it cannot be put in private